content/en/deployment/overview.md

```diff
@@ -8,3 +8,130 @@ categories: ["Canary Analysis", "Features", "Concepts"]
+      beforeDeployment: # a list of constraints that must be satisfied before the deployment to the target can begin 
+        - pause: # a pause constraint that will pause the execution of the deployment until an authorized user gives an approval
+            untilApproved: true
+```
+### Deployment Manifests
+
+Deploying your code with Armory CD-as-a-Service requires telling the CD-as-a-Service platform what artifacts should be 
+deployed. To accomplish this 
+you need to provide your Kubernetes manifests to Armory. 
+
+Manifests can be deployed to all environments or specific environments using a simple configuration: 
+
+```yaml
+manifests:
+  - path: manifests/your-manifest-1.yaml # this manifest will be deployed to all deployment targets defined in the targets block
+  - path: manifests/your-manifest-1.yaml # this manifest will be deployed to the 'target-2' target defined in the targets block
+    targets: ["target-2"]
+```
+             
+Manifest paths are relative to the directory from which you run the `armory deploy` command.
+
+### Deployment Strategies 
+A deployment strategy is the method by which your changes are deployed to a target. Strategies can use different techniques
+to allow for rapid rollback should a problem be discovered, minimizing the impact of potential issues to a small subset of users, 
+or can be optimized for speed. 
+
+Armory CD-as-a-Service offers two distinct deployment strategies that each have various trade-offs.
+
+For more details as to what deployment strategies are and how they differ, please see our [overview on the subject]({{< ref "deployment/strategies/overview" >}})
+
+#### The Canary Strategy
+A canary deployment involves releasing a new software version to a small subset of users or systems while leaving 
+the majority on the current version. This strategy allows for real-world testing and monitoring of the new version's performance
+and stability. If the canary users experience positive results, the new version can be gradually rolled out to a wider 
+audience.
+
+For more details on Canary deployments and how to configure them, you can reference the [canary strategy documentation]({{< ref "deployment/strategies/canary" >}}) 
+              
+#### The Blue-Green Strategy
+In a blue-green deployment, two identical environments, blue and green, are maintained. The blue 
+environment represents the current production version, while the green environment represents the new version being 
+deployed. The new version is deployed and tested in the green environment, and once validated, traffic is routed to green instead of blue. This strategy minimizes downtime and provides a quick rollback option if issues arise.
+
+For more details on blue-green deployments and how to configure them, you can reference the [blue-green strategy documentation]({{< ref "deployment/strategies/blue-green" >}})
```

Comment:
Try not to use #### (which doesn't render in the right menu).
I'd use a bullet point list here. (I did the snippet in VSCode but can't guarantee the formatting pasted correctly)

```
CD-as-a-Service offers two distinct deployment strategies:

* Canary

   A canary deployment involves releasing a new software version to a small subset of users or systems while leaving the majority on the current version. This strategy allows for real-world testing and monitoring of the new version's performance and stability. If the canary users experience positive results, the new version can be gradually rolled out to a wider audience.

* Blue/Green

   In a blue/green deployment, you maintain two identical environments, blue and green. The blue environment represents the current production version, while the green environment represents the new version that is being deployed. The new version is deployed and tested in the green environment, and once validated, CD-as-a-Service routes traffic is to green instead of blue. This strategy minimizes downtime and provides a quick rollback option if issues arise.


For more details as to what deployment strategies are and how they differ, see ({{< linkWithTitle "deployment/strategies/overview.md" >}}).
```

I removed the direct links to each strategy page since it's probably better to drive people directly to the strategy overview page.